frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/selected-nodes-table.tsx

```diff
@@ -81,6 +81,7 @@ export const SelectedNodesTable: React.FC<SelectedNodesTableProps> = ({ data })
         data={data}
         Rows={getRows}
         virtualize={false}
+        loaded
```

Comment:
This is a bug. Table was always in loading state since we are not using List Page component.
It should be always "loaded"